### PR TITLE
Wikibase: Layout and label changes to schema alignment tab

### LINKED ISSUE #5346: UI edits to schema selection and edits saving in Wikibase extension
This has been mentioned in previous PRs and issues, but starting a new issue to track all the final adjustments needed in more detail. This is a request to improve the following UI features:
Screenshot 2022-10-16 at 21 45 24

1. I think this text is not needed - it's too clunky in the layout; it's information experienced users know and don't need to see at all times; it's also something new users should learn from tutorials / documentation; new users (as shown during user testing) tend not to read small text anyway, and just click around until they find the solution to go forth, or just go and watch a tutorial;
2. Save schema should be called "Save schema progress".
3. Save schema progress / Discard changes should move below the main schema editing area (see wireframes).
4. This should probably just say "Select..."
5. "Save new" is potentially confusing if users just want to save and override an existing schema. Another idea was to keep the button to say only "Save", but I think this will make it potentially confusing next to the dropdown in case users decide they are just saving their selection choice. I propose we call this button "Save schema edits" and we move it to the right of the screen (see wireframes). 

### Proposed solution
Proposed redesign:
Screenshot 2022-10-16 at 23 59 48

For more context see complete design wireframes accessible via this link: https://docs.google.com/document/d/1uWkrid1UJwM3FCak_fI4VUY2EUabVUUFk9z6OeWtzKs/edit?usp=sharing

### Alternatives considered
Several previous iterations can be seen in the wireframe versions in the link above. The naming of the "Save" button and the schema vs schema template was discussed in more detail in previous PRs, e.g. https://github.com/OpenRefine/OpenRefine/pull/5303

### Additional context
The need to adjust this was discussed during the final round of user testing for the Commons extension project carried out Sept / Oct 2022.

### PR DESCRIPTION
**Pull request by Abbe98**
Fixes #5346

### CODE REVIEW

**Comment by Abbe98** [comment]
After:

Screenshot 2023-06-19 at 09-23-22 SHM - Fältinsamling - OpenRefine

**Review by lozanaross — APPROVED** [review]
Great, thanks for fixing this issue!

File: extensions/wikibase/module/langs/translation-en.json [hunk 1]
```diff
@@ -34,9 +34,8 @@
     "wikibase-addition/version-error": "Manifest version $1 is not supported. OpenRefine supports manifest versions 1.x and 2.x only.",
     "wikibase-schema/dialog-header": "Align to Wikibase",
     "wikibase-schema/target-wikibase-instance": "Target Wikibase instance:",
-    "wikibase-schema/dialog-explanation": "The schema below specifies how your tabular data will be transformed into <a href=\"$1\">$2</a> edits.",
     "wikibase-schema/start-from-an-existing-schema": "Start from an existing schema:",
-    "wikibase-schema/save-new-schema": "Save new...",
+    "wikibase-schema/save-new-schema": "Save schema edits",
```
Comment (wetneb) [inline extensions/wikibase/module/langs/translation-en.json:38]: "schema edits" seems confusing in this context: that could also refer to "the Wikibase edits generated by the schema". What about "schema changes"?

Comment (Abbe98) [reply extensions/wikibase/module/langs/translation-en.json:38]: Sound good to me, @lozanaross?

**Comment by wetneb** [comment]
To me, the labels of the buttons "Save schema edits" (now called "Save schema changes") and "Save schema" should be swapped to match their actual behavior.

**Pull request merged**